pkg/storage/stores/shipper/testutil/testutil.go

```diff
@@ -20,7 +20,7 @@ import (
 	chunk_util "github.com/grafana/loki/pkg/storage/chunk/util"
 )
 
-func AddRecordsToDB(t *testing.T, path string, dbClient *local.BoltIndexClient, start, numRecords int, bucketName []byte) {
+func AddRecordsToDB(t testing.TB, path string, dbClient *local.BoltIndexClient, start, numRecords int, bucketName []byte) {
```

Comment:
do we need `testing.TB` here?

Reply:
Yes, since we use it with both tests and benchmarks.